PINCache/PINMemoryCache.m

```diff
@@ -497,10 +502,14 @@ - (void)enumerateObjectsWithBlock:(PINMemoryCacheObjectBlock)block
         return;
     
     [self lock];
+        NSDate *now = [[NSDate alloc] init];
         NSArray *keysSortedByDate = [_dates keysSortedByValueUsingSelector:@selector(compare:)];
         
         for (NSString *key in keysSortedByDate) {
-            block(self, key, _dictionary[key]);
+            // If the cache should behave like a TTL cache, then only fetch the object if there's a valid ageLimit and  the object is still alive
+            if (!self->_ttlCache || self->_ageLimit <= 0 || fabs([[_dates objectForKey:key] timeIntervalSinceDate:now]) < self->_ageLimit) {
```

Comment:
@garrettmoon well I think we're all set here then. The getters have been pushed, and I'm not sure how mark/resolve comments in github. Maybe you can't.